nerfstudio/cameras/cameras.py

```diff
@@ -747,8 +747,6 @@ def _compute_rays_for_vr180(
 
             rotation = c2w[..., :3, :3]
 
-            -torch.pi * ((x - cx) / fx)[0]
```

Comment:
Is this a typo?

Reply:
not sure what this line was doing there... the value is not assigned to anything, so I deleted it.